### PR TITLE
CDD-1888: Removed old file upload endpoint.

### PR DESCRIPTION
**Pull request by rashadmughal**

### CODE REVIEW

File: test/unit/controllers/FileUploadUpscanNotificationControllerSpec.scala [hunk 1]
```diff
@@ -28,32 +28,33 @@ import play.api.libs.json.JsValue
...
+class FileUploadUpscanNotificationControllerSpec extends PlaySpec with MockitoSugar with Eventually {
 
   trait SetUp {
-    val mockNotificationService = mock[BatchFileUploadNotificationService]
+    val mockNotificationService = mock[FileUploadNotificationService]
     val mockToXmlNotification = mock[UpscanNotificationCallbackToXmlNotification]
     val mockErrorToXmlNotification = mock[InternalErrorXmlNotification]
-    val mockBusinessService = mock[BatchFileUploadUpscanNotificationBusinessService]
+    val mockBusinessService = mock[FileUploadUpscanNotificationBusinessService]
     val mockCdsLogger = mock[CdsLogger]
```
Comment (paulblakehmrc) [inline test/unit/controllers/FileUploadUpscanNotificationControllerSpec.scala]: don't need `mockCdsLogger` now

File: README.md [hunk 2]
```diff
@@ -155,35 +155,6 @@ that `default` configuration is declared directly inside the `customs-declaratio
       "url": "http://currenturl/customs-declarations"
```
Comment (googley42) [inline README.md]: We have File Upload related schemas in 2 places - eg for version `2.0`
```
public/api/conf/2.0/schemas/wco/declaration
public/api/conf/2.0/schemas/wco/fileupload
```

Comment (rashadmughal) [reply README.md]: moved to fileupload for version 2 and 3

**Comment by paulblakehmrc** [comment]
:+1:

**Pull request merged**